utils/constants.py

```diff
@@ -7,3 +7,17 @@
 
 # maximum number of players that can be added
 MAX_NUM_PLAYERS_TEAM = 10
+
+TIER_VALUE = {
+    "IRON": 1,
+    "BRONZE": 2,
+    "SILVER": 3,
+    "GOLD": 4,
+    "PLATINUM": 5,
+    "DIAMOND": 6,
+    "MASTER": 7,
+    "GRANDMASTER": 8,
+    "CHALLENGER": 9,
```

Comment:
unranked?

Reply:
oh true

Reply:
I have to consider unranked, master, grandmaster and challenger since they dont have roman numers at the end. I think I might need to change some stuff from add summoner and list summoner as well. Forgot about those conditions and just realized.

Reply:
I think Sona Bot generalizes unranked players as SILVER 4 level, should I follow that?